kuma/wiki/kumascript.py

```diff
@@ -298,7 +299,11 @@ def macro_sources():
     response = requests.get(ks_macro_url)
     if response.status_code == 200:
         macros_raw = response.json()['macros']
-        return {md['name']: md['filename'] for md in macros_raw}
+        # Ensure Normal Form C used on GitHub
+        return {
+            unicodedata.normalize('NFC', md['name']):
```

Comment:
I didn't either, until I hit issues exporting [historical macros to files](https://github.com/mdn/archived_kumascript/blob/master/export_templates.py#L162-L176).